clustering/spi/src/main/java/org/wildfly/clustering/spi/CacheGroupServiceName.java

```diff
@@ -34,10 +34,10 @@ public ServiceName getServiceName(String container, String cache) {
             return GroupServiceName.NODE_FACTORY.getServiceName(container).append(cache);
         }
     },
-    GROUP() {
+    GROUP_CACHE() {
```

Comment:
This name makes no sense - this value provides the service name of a org.wildfly.clustering.group.Group object.  This is not a cache.